serverless/config/log.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2021 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package config provides the descriptor structs and example configs for
+// the different entities. This allows for a common description of logs,
+// witnesses, etc.
+package config
+
+import (
+	"errors"
+	"fmt"
+	"net/url"
+)
+
+type Log struct {
```

Comment:
This probably should have a struct comment on it since it's exported?

Reply:
Done.